fe/fe-core/src/main/java/org/apache/doris/analysis/ShowCreateRoutineLoadStmt.java

```diff
@@ -0,0 +1,73 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.analysis;
+
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.qe.ShowResultSetMetaData;
+
+// SHOW CREATE ROUTINE LOAD statement.
+public class ShowCreateRoutineLoadStmt extends ShowStmt {
+    
+    private static final ShowResultSetMetaData META_DATA =
+            ShowResultSetMetaData.builder()
+                    .addColumn(new Column("Routine Load", ScalarType.createVarchar(20)))
+                    .addColumn(new Column("Create Routine Load", ScalarType.createVarchar(30)))
+                    .build();
+    
+    private final LabelName labelName;
+    
+    public ShowCreateRoutineLoadStmt(LabelName labelName) {
+        this.labelName = labelName;
+    }
+    
+    public String getDb() {
+        return labelName.getDbName();
+    }
+    
+    public String getLabel() {
+        return labelName.getLabelName();
+    }
+    
+    @Override
+    public void analyze(Analyzer analyzer) throws AnalysisException {
+        if (labelName == null) {
```

Comment:
Label_name in the grammar file is not optional, so it will not be empty.

Reply:
OK, thanks for the warning